controllers/topics.controllers.js

```diff
@@ -17,4 +17,16 @@ exports.getArticleById = (req, res, next) => {
       res.status(200).send({ article });
     })
     .catch(next);
-};
\ No newline at end of file
+};
+
+exports.getAllArticles = (req, res, next) => {
+
+  fetchAllArticles()
+    .then((articles) => {
+      const newArticles = articles.map(x => Object.assign({}, x, {
```

Comment:
Is this to convert your comment_count to a number? If so, this should be handled in your SQL query. Have a look at using CAST!

Reply:
resolved